### ISSUE TITLE
SciHub requiring keyword values to be within quotes

### ISSUE THREAD

**Issue by radosuav** [sentinelsat/sentinelsat#383]
It appears that there was a subtle change to SciHub interface recently. Previously I could specify extra keywords using single quotes, e.g.:
```
other_search_keywords["orbitdirection"] = "descending"
other_search_keywords["timeliness"] = "Near Real Time"
```
Now they have to be in double quotes (i.e. ' followed by "):
```
other_search_keywords["orbitdirection"] = '"descending"'
other_search_keywords["timeliness"] = '"Near Real Time"'
```
Otherwise apihub never responds to the query ("timeliness") or ignores the extra keyword ("orbitdirection").

There are some keywords which still work with single quotes, e.g. 
`other_search_keywords["productlevel"] = "L2"`
seems to work fine.

To be on the safe side, maybe all keywords should be put within extra quotes.

**Comment by valgur** [comment]
Interesting. We already take care to handle spaces in values correctly (#169), but something might have changed on the server side. I will try to take a closer look at this soon.

**Comment by valgur** [comment]
Fixed by #390.
